packages/components/boxDetailTeriroy/BoxDetailTeriroy.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
```

Comment:
Why this name ? `BoxDetailTeriroy` is for what ? Do you mean `BoxDetailTeritori` ?